app/web/src/components/Ai/GenerateAwsAssetSchemaPanel.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <div class="flex flex-row gap-xs place-items-end">
+    <VormInput
+      ref="cli"
```

Comment:
These refs do not appear to be in use - was this just pattern matching?

Reply:
Yep! I'll try and remove them and if it compiles, win!